models/engine/file_storage.py

```diff
@@ -68,3 +68,20 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """retrieve one object"""
+        if cls.__name__ in classes and id is not None:
+            objs = self.all(cls.__name__)
+            for key in objs.keys():
+                if key == cls.__name__ + '.' + id:
+                    return objs[key]
+        return None
```

Comment:
You can optimize all this part +74 to +79 using the .`get(key)` dictionary function, where `key ` is class name + object id
```suggestion
        def get(self, cls, id):
            clss = self.all(cls)
            key = cls + '.' + id
            return clss.get(key)
```